purchases/src/main/kotlin/com/revenuecat/purchases/ads/events/AdTracker.kt

```diff
@@ -0,0 +1,71 @@
+package com.revenuecat.purchases.ads.events
+
+import com.revenuecat.purchases.InternalRevenueCatAPI
+import com.revenuecat.purchases.common.events.EventsManager
+
+@InternalRevenueCatAPI
+class AdTracker internal constructor(
```

Comment:
This is kinda the public API I was thinking. This would be called by users like:
```
Purchases.sharedInstance.adTracker.trackAdDisplayed(....)
```

I thought about having a single "event" object that users need to build, but feels like extra complexity and the API can just flatten it out. Wdyt?